### PR TITLE
[YUNIKORN-1297] move constants to md file

### PR DESCRIPTION
**Pull request by wilfred-s**
### What is this PR for?
The md file is the source of truth moving the constants to the md file
and run a make to re-generate the go file.
Run go fmt over the file to make sure it is properly formatted.

### What type of PR is it?
* [X] - Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-1297

### How should this be tested?
Follow up jira is logged to change the build process and fail the process if this happens again

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -91,23 +93,25 @@ $(SI_PROTO).tmp: $(SI_SPEC)
...
+	go fmt $@
 	(diff $@ $(CONSTANTS_GO) > /dev/null 2>&1 || mv -f $@ $(CONSTANTS_GO)) && \
 		rm -f $@
 
-$(INTERFACE_GO).tmp: $(SI_SPEC)
+$(INTERFACE_TMP): $(SI_SPEC)
 	test -d $(API_LIB) || mkdir -p $(API_LIB)
 	@echo "$$GENERATED_HEADER" > $@
-	cat $< | sed -n -e '/``golang$$/,/^```$$/ p' | sed '/^```/d' >> $@
+	cat $< | sed -n -e '/``golang$$/,/^```$$/ p' | sed '/^```/ s/.*//g' >> $@
```
Comment (HuangTing-Yao) [inline Makefile:108]: Why we got ``golang$$, not ```golang$$$ which with three grave accent?
The constants with three grave accent above.

Comment (wilfred-s) [reply Makefile]: I thought I had fixed that one too, looks like I missed it.

**Review by HuangTing-Yao — APPROVED** [review]
LGTM , Thanks for fixing this.

**Comment by manirajv06** [comment]
looks good.

**Review by manirajv06 — APPROVED** [review]

**Pull request closed without merging**